java/code/src/com/redhat/rhn/common/localization/LocalizationService.java

```diff
@@ -693,6 +693,10 @@ public Collator newCollator() {
      */
     private static Locale getDefaultLocale() {
         String defaultLocale = ConfigDefaults.get().getDefaultLocale();
+        // Set fallback value if default can't be read from the config
```

Comment:
I agree with the spirit of the patch but would suggest a different implementation, that is to add the default value as second parameter in `ConfigDefaults.getDefaultLocale()`:

https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/redhat/rhn/common/conf/ConfigDefaults.java#L1023

Reason is just consistency with other cases/centralization, see for example:

https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/redhat/rhn/common/conf/ConfigDefaults.java#L933